components/datatypes/financial-account.schema.json

```diff
@@ -33,9 +35,7 @@
         "xdm:financialAccountOwner": {
           "title": "Financial Account Owners",
           "type": "array",
-          "items": {
-            "$ref": "https://ns.adobe.com/xdm/context/profile-personal-details"
-          },
+          "$ref": "https://ns.adobe.com/xdm/context/profile-personal-details",
```

Comment:
Did you remove "items" here ?

Reply:
This is still unresolved. We need to add "items" here

Reply:
I believe this is outdated and items does exist in the current schema.